README.md

```diff
@@ -4,6 +4,7 @@ This is a programmatic interface in Ruby for SignalFx's metadata and
 ingest APIs. It is meant to provide a base for communicating with
 SignalFx APIs that can be easily leveraged by scripts and applications
 to interact with SignalFx or report metric and event data to SignalFx.
+You will need a SignalFx account and an access token to use them.
```

Comment:
I don't get why you are starting to add this to the docs. It's pretty much true of every page in every doc we produce. Did someone ask you to add this?

Reply:
just copy pasting from other libraries. i can remove